FlaskApp/app/templates/barchart.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% block extra_head %}
+<title>Realtime Stock chart</title>
+{% endblock extra_head %}
+{% block menu %}
+<div class="container">
+    <div class="header clearfix">
+        <nav>
+            <ul class="nav nav-pills pull-right">
+                <li role="presentation" class="active"><a href="#">Home</a></li>
+                <li role="presentation"><a href="#">About</a></li>
+                <li role="presentation"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <h3 class="text-muted">Stock Prediction</h3>
+    </div>
+</div>
+{% endblock menu %}
+
+{% block content %}
+<div class="container">
+    <div class="jumbotron">
+        <p>This project aims to predict stock using data collected from Twitter and Facebook. The prediction and real time value are shown as below.</p>
+    </div>
+    <div class="panel">
+
+        <div class="chart" id="container"></div>
+    </div>
+    <table id="datatable" >
+        <thead>
+        <tr>
+            <th></th>
```

Comment:
@hyperchi  well, Zach wants me to show how many tweets mentioned the three companies we care about separately and display them as a bar chart. So I created a new page for this.